homeassistant/components/dublin_bus_transport/sensor.py

```diff
@@ -92,18 +81,12 @@ def state(self):
     def device_state_attributes(self):
         """Return the state attributes."""
         if self._times is not None:
-            next_up = "None"
-            if len(self._times) > 1:
-                next_up = f"{self._times[1][ATTR_ROUTE]} in "
-                next_up += self._times[1][ATTR_DUE_IN]
-
             return {
-                ATTR_DUE_IN: self._times[0][ATTR_DUE_IN],
-                ATTR_DUE_AT: self._times[0][ATTR_DUE_AT],
                 ATTR_STOP_ID: self._stop,
+                ATTR_DUE_IN: self._times[0][ATTR_DUE_IN],
```

Comment:
This is the same as the state, there is no need for it to be an attribute as well.

Reply:
fair point. thanks